From d5cd3a824cd58b2b9eec31a0407c23fa39540386 Mon Sep 17 00:00:00 2001
From: fujr <fjrcn@outlook.com>
Date: Mon, 27 Feb 2023 19:52:22 +0800
Subject: [PATCH] add gl config

---
 .../boot/dts/mediatek/mt7981-360-t7-base.dtsi | 30 +++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/target/linux/mediatek/files-5.4/arch/arm64/boot/dts/mediatek/mt7981-360-t7-base.dtsi b/target/linux/mediatek/files-5.4/arch/arm64/boot/dts/mediatek/mt7981-360-t7-base.dtsi
index 92a704e5e2..1f3ce6da9e 100644
--- a/target/linux/mediatek/files-5.4/arch/arm64/boot/dts/mediatek/mt7981-360-t7-base.dtsi
+++ b/target/linux/mediatek/files-5.4/arch/arm64/boot/dts/mediatek/mt7981-360-t7-base.dtsi
@@ -12,6 +12,29 @@
 		led-upgrade = &green_led;
 	};
 
+
+	gl-hw {
+                compatible = "gl-hw-info";
+                model = "mt3000";
+                wan = "eth0";
+                lan = "eth1";
+                temperature = "/sys/devices/virtual/thermal/thermal_zone0/temp";
+                switch-button = "gpio-455";
+                reset-button = "gpio-456";
+                radio = "mt798111 mt798112";
+                cfg-partition = "/dev/mtd7";
+                flash_size = <128>;
+		dfs;
+                factory_data {
+                        device_mac = "stock-factory", "0xb0";
+                        device_ddns = "stock-factory", "0xc0";
+                        device_sn_bak = "stock-factory", "0xd0";
+                        device_sn = "stock-factory", "0xe0";
+                        country_code = "stock-factory", "0x7a";
+                };
+        };
+
+
 	chosen {
 		bootargs = "console=ttyS0,115200n1 loglevel=8  \
 				earlycon=uart8250,mmio32,0x11002000";
@@ -96,6 +119,13 @@
 	mdio: mdio-bus {
 		#address-cells = <1>;
 		#size-cells = <0>;
+		ethernet-phy@0 {
+            compatible = "ethernet-phy-id03a2.9461";
+            reg = <0x00>;
+            phy-mode = "gmii";
+            nvmem-cells = <0x12>;
+            nvmem-cell-names = "phy-cal-data";
+        	};
 	};
 };
 
-- 
2.25.1

